### PR TITLE
Add test codes and set dynamic configurable parameters

### PR DESCRIPTION
**Pull request by Holt2016**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: edgemesh/pkg/common/utils_test.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package common
+import (
+	"testing"
+)
+
+func TestSplitServiceKey(t *testing.T) {
+	SplitServiceKey("name")
```
Comment (anyushun) [inline edgemesh/pkg/common/utils_test.go]: Unit testing consists of at least three steps:
1. Construct test data
2. Execute the tested method
3. Verify test results

You only made 1 and 2

File: edgemesh/pkg/registry/registry_test.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+"testing"
+)
+
+func TestFindMicroServiceInstances(t *testing.T) {
+	c:= context.GetContext(context.MsgCtxTypeChannel)
+	sd := &ServiceDiscovery{
+		metaClient:client.New(c),
+	}
+
+	_, _ = sd.FindMicroServiceInstances("123", "name.namespace", utiltags.Tags{})
```
Comment (anyushun) [inline edgemesh/pkg/registry/registry_test.go]: Check execute result please

File: edgemesh/pkg/server/tcp.go [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
-	"net"
-	"net/http"
-
 	"github.com/go-chassis/go-chassis/core/common"
 	"github.com/go-chassis/go-chassis/core/handler"
 	"github.com/go-chassis/go-chassis/core/invocation"
 	"github.com/kubeedge/beehive/pkg/common/config"
 	"github.com/kubeedge/beehive/pkg/common/log"
 	"github.com/kubeedge/kubeedge/edgemesh/pkg/resolver"
+	"io"
```
Comment (anyushun) [inline edgemesh/pkg/server/tcp.go]: The order of the golang import package is:
```
Golang package

3rd package

Custom package
```

so ```io```,```io/util```,```net```,```net/http``` should before ```github.com/xxx```

File: edgemesh/pkg/server/tcp.go [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
+	"io/ioutil"
+	"net"
+	"net/http"
 )
 
 func StartTCP() {
-	server := config.GetString("server", "0.0.0.0")
-	port := config.GetString("port", "8080")
+	server, _ := getIP()
```
Comment (anyushun) [inline edgemesh/pkg/server/tcp.go]: Check err returned by getIP please, if err is not nil, refer to the startDnsServer function

**Comment by rohitsardesai83** [comment]
why is it showing so many commits ?

**Comment by rohitsardesai83** [comment]
please resolve conflicts as well.

**Pull request closed without merging**